src/main/resources/application.properties

```diff
@@ -1,2 +1,3 @@
 logging.file.name=logs/ecomapp.log
 logging.level.root=INFO
+com.startsteps.ecommerceapi=${JWT_SECRET}
```

Comment:
it would be good if you used a different name for a secret. How would I know if this is a JWT secret by just looking at the name